.github/workflows/_build.yml

```diff
@@ -103,7 +99,6 @@ jobs:
         if: steps.avd-cache.outputs.cache-hit != 'true'
         uses: reactivecircus/android-emulator-runner@v2
         with:
-          target: google_apis
```

Comment:
stopped using the google apis version as we don't need it and the non-google one is meant to be more stable. i tried this with the old api level, but the emulator wasn't available